stdlib/tkinter/__init__.pyi

```diff
@@ -124,6 +125,8 @@ _ScreenUnits = Union[str, float]  # manual page: Tk_GetPixels
 _XYScrollCommand = Union[str, Callable[[float, float], Any]]  # -xscrollcommand and -yscrollcommand in 'options' manual page
 _TakeFocusValue = Union[int, Literal[""], Callable[[str], Optional[bool]]]  # -takefocus in manual page named 'options'
 
+_ObjectId = NewType("ObjectId", int)
```

Comment:
```suggestion
_ObjectId = NewType("_ObjectId", int)
```

Reply:
I would rename this to `_CanvasItemId`:
- There are lots of classes in this file, but only canvases use `int` IDs.
- The canvas docs say "item IDs".

Reply:
While `_CanvasObjectId` is slightly better, I think `_CanvasItemId` would be even better, for consistency with [the docs](http://tcl.tk/man/tcl8.7/TkCmd/canvas.htm#M16).